src/backend/src/prisma/seed-data/teams.seed.ts

```diff
@@ -90,4 +93,15 @@ const financeTeam = (headId: number): Prisma.TeamCreateArgs => {
   };
 };
 
-export const dbSeedAllTeams = { ravens, orioles, justiceLeague, huskies, avatarBenders, plLegends, financeTeam };
+const meanGirls = (headId: number): Prisma.TeamCreateArgs => {
```

Comment:
Great choice of characters, but can the name of the team relate to Slack so it's more clear what it can be used for?